commit 00eb12a2c768cae3ca136110baacb5a35e9066a8
Author: Gary Lin <glin@suse.com>
Date:   Mon Jun 25 18:31:24 2018 +0800

    BaseTools: Fix a typo in ini.py
    
    "if mis not None:" => "if m is not None:"
    
    Contributed-under: TianoCore Contribution Agreement 1.1
    Cc: Jaben Carsey <jaben.carsey@intel.com>
    Cc: Yonghong Zhu <yonghong.zhu@intel.com>
    Cc: Liming Gao <liming.gao@intel.com>
    Signed-off-by: Gary Lin <glin@suse.com>
    Reviewed-by: Jaben Carsey <jaben.carsey@intel.com>

diff --git a/BaseTools/Scripts/PackageDocumentTools/plugins/EdkPlugins/basemodel/ini.py b/BaseTools/Scripts/PackageDocumentTools/plugins/EdkPlugins/basemodel/ini.py
index bf1040d6ba..ea83327052 100644
--- a/BaseTools/Scripts/PackageDocumentTools/plugins/EdkPlugins/basemodel/ini.py
+++ b/BaseTools/Scripts/PackageDocumentTools/plugins/EdkPlugins/basemodel/ini.py
@@ -122,7 +122,7 @@ class BaseINIFile(object):
                 continue
 
             m = section_re.match(templine)
-            if mis not None: # found a section
+            if m is not None: # found a section
                 inGlobal = False
                 # Finish the latest section first
                 if len(sObjs) != 0:
